pkg/component/component.go

```diff
@@ -452,6 +452,15 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 	}
 	targetPath := fmt.Sprintf("%s/src", s2iSrcPath)
 
+	stdin := bytes.NewBuffer([]byte{})
+	stdout := bytes.NewBuffer([]byte{})
+	stderr := bytes.NewBuffer([]byte{})
```

Comment:
You can use this instead:
```sh
stdin, stdout, stderr := bytes.NewBuffer([]byte{}), bytes.NewBuffer([]byte{}), bytes.NewBuffer([]byte{})
```

May look a bit nicer :+1:

Reply:
PR modified. Please revisit the PR @cdrage